CREDITS.md

```diff
@@ -0,0 +1,46 @@
+**CREDITS**
+
+Below is a list of people which contributions in the community have be helpful for this repository to progress.
+
+**Per Bering**: One of the first promoters of Docker for Sitecore. He is a major contributor to this repository, and actively helps peers with Docker related matters on different channels.
+https://github.com/sitecoreops/sitecore-images
+https://github.com/Sitecore/docker-demo
+https://github.com/pbering/SitecoreContainers
+https://github.com/sitecoreops/sitecore-images
+https://github.com/pbering/sitecore-nine-docker
+https://github.com/pbering/SockerVS2017
+https://github.com/pbering/solrwin
+
+**Joost Meijles (Aviva Solutions)**: His contributions in GitHub have been a source of inspiration. He is also active in community discussions about using Docker with Sitecore.
```

Comment:
Please:
- add https://github.com/avivasolutionsnl/sitecore-commerce-docker iso https://github.com/joostmeijles/sitecore-commerce-docker
- remove my personal fork (and not maintained): https://github.com/joostmeijles/sitecore-docker

Reply:
Done